rust/hyperlane-base/src/settings/trace/mod.rs

```diff
@@ -63,7 +63,7 @@ impl TracingConfig {
     pub fn start_tracing(&self, metrics: &CoreMetrics) -> Result<console_subscriber::Server> {
         let mut target_layer = Targets::new().with_default(self.level);
 
-        if self.level < Level::DependencyTrace {
+        if self.level == Level::Trace {
             // Reduce log noise from trusted libraries that we can reasonably assume are working correctly
             target_layer = target_layer
                 .with_target("hyper", Level::Info)
```

Comment:
lmaooo this is totally the one that's at fault

From https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/targets/struct.Targets.html#filtering-with-targets:
> A Targets filter consists of one or more [target](https://docs.rs/tracing-core/0.1.32/x86_64-unknown-linux-gnu/tracing_core/metadata/struct.Metadata.html#method.target) prefixes, paired with [LevelFilter](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.LevelFilter.html)s. If a span or event’s [target](https://docs.rs/tracing-core/0.1.32/x86_64-unknown-linux-gnu/tracing_core/metadata/struct.Metadata.html#method.target) begins with one of those prefixes, and its [level](https://docs.rs/tracing-core/0.1.32/x86_64-unknown-linux-gnu/tracing_core/metadata/struct.Level.html) is at or below the [LevelFilter](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.LevelFilter.html) enabled for that prefix, then the span or event will be enabled.

`hyper` prefix is conflicting with some of our targets that start with `hyperlane_base` and `hyperlane_core` 🤦‍♂️ 

I wonder if we can change this to `hyper::` and that'd fix it? And given that I wonder if we should just stick with `if self.level < Level::DependencyTrace`